Sources/SwiftSyntax/SyntaxRewriter.swift.gyb

```diff
@@ -91,7 +91,11 @@ open class SyntaxRewriter {
       if let newNode = visitAny(node._syntaxNode) { return newNode }
       return Syntax(visit(node))
 %   else:
+%     if node.must_uphold_invariant:
+      let node = ${node.name}(data)!
```

Comment:
Could you add a comment like the following here to document why the use of the `!` is safe here?

```swift
// We know that the SyntaxData is valid since we are walking a valid syntax tree and haven't modified the syntax data.  Thus the below initialiser will never return nil.
```

Reply:
Done.

Reply:
I think you forgot to add the comment ;-)

Reply:
Sorry about that. Fixed.